types/nationalAvalancheCenter/schemas.ts

```diff
@@ -53,18 +53,20 @@ export const forecastPeriodSchema = z.nativeEnum(ForecastPeriod);
 
 export const mediaTypeSchema = z.nativeEnum(MediaType);
 
-export const mediaLinksSchema = z.object({
-  large: z.string(),
-  medium: z.string(),
-  original: z.string(),
-  thumbnail: z.string(),
-});
+export const mediaLinksSchema = z
+  .object({
+    large: z.string().optional(),
+    medium: z.string().optional(),
+    original: z.string().optional(),
+    thumbnail: z.string().optional(),
+  })
+  .or(z.string()); // when this field is not populated, it's an empty string, not a null
```

Comment:
@floatplane would like your input on how to best handle this ... what a mess.

Reply:
My current best thought is to fix up issues post-fetch, but pre-validation: https://github.com/stevekuznetsov/avalanche-forecast/blob/c27a6dee43269e0097a3526d5b8678f4796773dc/hooks/useAvalancheForecast.ts#L23-L33

For this particular case, I guess I'd look for empty strings and turn them into nulls to make it easier to deal with?

Reply:
ignore this, I didn't realize you had already gone down a different road with `transform`

Reply:
Sorry! Forgot to mark this resolved